### ISSUE TITLE
Mixers have no adjustment function

### ISSUE THREAD

**Issue by prondel** [FrSky-OS/discussion#14]
This is a must have for a radio like the horus. Any mixer should propose in addtion to the switch to activate or deactivate it, a way to modify the mixing rate with a trim or a knob.

**Comment by AloftHobbies** [comment]
Yes, this is a good point. To illustrate, if you are flying a plane for the first time, you can set up a flap to elevator compensation mix. You simply never know how your plane is going to respond when flaps are applied. Some planes will climb, others will descend, so we need to mix in the correct amount of elevator.  Modern radios allow you to use an input to adjust this elevator mix in flight. So now when we test fly our plane, we can deploy the flaps, see if the plane needs more or less elevator, simply adjust the mix via whatever input that we picked. 

This is a great feature for most all mixes and differential. It should be an option that can be turned on or off.
